README.md

```diff
@@ -1 +1,27 @@
-# ancestry
\ No newline at end of file
+# ancestry
+
+Prioritising and assigning ancestry to populations.
+
+## Running hail
+
+To connect  to hail with the required gnomAD dependencies, run the following command:
+
+```hailctl dataproc start --max-age 2h --region australia-southeast1 my-cluster --packages gnomad```
+
+Some commands are computationally expensive and require more workers. For these jobs, you can modify the cluster. I specify 100 preemptible workers with the following command:
+
+```hailctl dataproc modify my-cluster --num-preemptible-workers 100 --num-workers 2```
```

Comment:
Fyi, you can get away with just using a single backtick when wanting in-line code in markdown:

```suggestion
`hailctl dataproc modify my-cluster --num-preemptible-workers 100 --num-workers 2`
```